web/src/pages/Analytics.tsx

```diff
@@ -0,0 +1,376 @@
+import { BsGearFill } from "react-icons/bs";
+import { HiBell, HiOfficeBuilding, HiRefresh } from "react-icons/hi";
+import { useRecoilValue } from "recoil";
+import useSWR from "swr";
+
+import { ConnectToSingleStoreButton } from "@/components/ConnectToSinglestoreButton";
+import { Loader } from "@/components/customcomponents/loader/Loader";
+import { EnableSimulatorButton } from "@/components/EnableSimulatorButton";
+import { Heatmap } from "@/components/HeatMap";
+import { SetupDatabaseButton } from "@/components/SetupDatabaseButton";
+import {
+  CustomerMetrics,
+  customerMetrics,
+  estimatedRowCountObj,
+  overallConversionRate,
+  ZoneMetrics,
+  zoneMetrics,
+} from "@/data/queries";
+import { connectionConfig, simulatorEnabled } from "@/data/recoil";
+import { useConnectionState } from "@/view/hooks/hooks";
+import { useSimulationMonitor } from "@/view/hooks/useSimulationMonitor";
+import { useSimulator } from "@/view/hooks/useSimulator";
+
+const formatPct = format(",.2%");
+const formatStat = format(".4~s");
+
+const NotificationZoneMap = () => {
+  const [isSmallScreen] = useMediaQuery("(max-width: 640px)");
+
+  return (
+    <Flex gap={5} direction={isSmallScreen ? "column" : "row"}>
+      <Stack flex={2}>
+        <StatGrid />
+      </Stack>
+      <Box
+        padding="17px"
+        flex={4}
+        borderRadius={10}
+        border="1px solid grey"
+        position="relative"
+      >
+        <Flex
+          direction="row"
+          justifyContent="space-between"
+          width="100%"
+          gap={10}
+          alignItems="center"
+        >
+          <Text fontWeight="bold" fontSize="sm" textTransform="uppercase">
+            Offer conversion rates by Notification zone
+          </Text>
+          <Flex width="30%" direction="column">
+            <Box width="100%">
+              <Progress
+                colorScheme="transparent"
+                height={2}
+                bgGradient="linear(to-r, rgba(127, 17, 224, 1), white)"
+                value={90}
+              />
+            </Box>
+            <Flex width="100%" justifyContent="space-between">
+              <Text>
+                <small>High</small>
+              </Text>
+              <Text>
+                <small>Low</small>
+              </Text>
+            </Flex>
+          </Flex>
+        </Flex>
+        <br />
+
+        <Heatmap
+          height={400}
+          useCells={useConversionCells}
+          colorInterpolater={interpolateBuPu}
+          getCellConfig={({ conversionRate, wktPolygon }: ZoneMetrics) => ({
+            value: conversionRate,
+            wktPolygon,
+          })}
+        />
+      </Box>
+    </Flex>
+  );
+};
+
+const DashboardContainerChild = () => {
+  const { initialized, connected } = useConnectionState();
+  const enabled = useRecoilValue(simulatorEnabled);
+  useSimulationMonitor(enabled);
+  useSimulator(enabled);
+
+  if (!connected) {
+    return <ConnectToSingleStoreButton />;
```

Comment:
as I said previsouly, I think this can be removed.

Reply:
removed and automated